### PR TITLE
Added tinting and sprite placement functionality

### PR DESCRIPTION
**Pull request by Ober3550**
I tried adding texture rotation but the method described by ocornut didn't work for some reason. For now here's just some extra functions to add color and positioning to sprites passed to the imgui library

### CODE REVIEW

**Comment by Ober3550** [comment]
Sorry for not submitting something that works ':D. I was testing it on a bigger project I already had installed and didn't want to just copy paste an older version of the project since you've made changes. Idk why but I can't get the project to compile
https://cdn.discordapp.com/attachments/410086810853703682/768309926191169576/unknown.png
Got any ideas on how to fix this?

**Comment by eliasdaler** [comment]
Well, you have added the same declarations for the overloads which already exist. The overload you've added on line 83 in imgui-SFML.h already exists on line 93

File: imgui-SFML.cpp [hunk 1]
```diff
@@ -549,6 +549,15 @@ void SetLStickYAxis(sf::Joystick::Axis lStickYAxis, bool inverted) {
 }  // end of namespace SFML
 
 /////////////// Image Overloads
+void Image(const sf::Sprite& sprite) {
+    sf::FloatRect bounds = sprite.getLocalBounds();
+    sf::Vector2f scale = sprite.getScale();
+    sf::Color tintColor = sprite.getColor();
+    sf::Color borderColor = sf::Color::Transparent;
```
Comment (eliasdaler) [inline imgui-SFML.cpp:556]: No need to create a variable and pass it, as it's a default argument for border color in an overload

File: imgui-SFML.cpp [hunk 2]
```diff
@@ -549,6 +549,15 @@ void SetLStickYAxis(sf::Joystick::Axis lStickYAxis, bool inverted) {
 }  // end of namespace SFML
 
 /////////////// Image Overloads
+void Image(const sf::Sprite& sprite) {
+    sf::FloatRect bounds = sprite.getLocalBounds();
+    sf::Vector2f scale = sprite.getScale();
+    sf::Color tintColor = sprite.getColor();
```
Comment (eliasdaler) [inline imgui-SFML.cpp:555]: No need to create a variable - just do:
```cpp
Image(sprite, ..., sprite.getColor());
```

File: imgui-SFML.cpp [hunk 3]
```diff
@@ -549,6 +549,15 @@ void SetLStickYAxis(sf::Joystick::Axis lStickYAxis, bool inverted) {
...
+    sf::Color borderColor = sf::Color::Transparent;
+    ImGui::SetCursorPos(ImVec2(sprite.getPosition().x - sprite.getOrigin().x * sprite.getScale().x, sprite.getPosition().y - sprite.getOrigin().y * sprite.getScale().y));
```
Comment (eliasdaler) [inline imgui-SFML.cpp:557]: I don't like how this gets repeated for all overloads - please make a helper function
Also, you can just do something like this:
```cpp
sf::Vector2f cursorPos = sprite.getPosition() - sprite.getOrigin() * sprite.getScale();
ImGui::SetCursorPos(ImVec2(cursorPos.x, cursorPos.y));
```

**Review by eliasdaler — CHANGES_REQUESTED** [review]
All the comments that I've left for one overload apply to the other overloads

**Pull request closed without merging**